products/models.py

```diff
@@ -0,0 +1,67 @@
+from django.db import models
+
+class Product(models.Model):
+    name         = models.CharField(max_length=50)
+    price        = models.DecimalField(max_digits=None, decimal_places=2)
+    discription  = models.TextField()
+    sub_category = models.ForeignKey('Sub_category', on_delete=models.CASCADE)
+
+    created_at   = models.DateTimeField(auto_now_add=True)
+
+    class Meta:
+        db_table = 'products'
+
+
+class Image(models.Model):
+    image_url = models.CharField(max_length=200)
+    product   = models.ForeignKey('Product', on_delete=models.CASCADE)
+
+    class Meta:
+        db_table = 'images'
+
+
+class Sub_category(models.Model):
```

Comment:
클래스명은 CapWords 컨벤션을 사용해주세요!
```suggestion
class SubCategory(models.Model):
```